lib/tlscontext.h

```diff
@@ -44,6 +44,16 @@ typedef enum
   TVM_REQUIRED=0x0020,
 } TLSVerifyMode;
 
+typedef enum
+{
+  TSO_NONE,
+  TSO_NOSSLv2=0x0001,
+  TSO_NOSSLv3=0x0002,
+  TSO_NOTLSv1=0x0004,
+  TSO_NOTLSv11=0x0008,
+  TSO_NOTLSv12=0x0010,
+} TLSSslOptions;
+
```

Comment:
hi,

as I said: this is consistent with the other parts from the same module in _this_ form.
I also said, that I agree with you about the typedef enum part.
So, I am wondering if
* you do not trust my word, that I will make a code cleaning for this module where I fix the ssl-options and the verify-mode both
* or you want a code where two very-very similar functions (tls_lookup_verify_mode, tls_lookup_options) are defined two different ways in the same code
* or you want a larger patch, where a code cleanup is done at the same branch where a new function/ option is implemented?